test/runtests.jl

```diff
@@ -4,6 +4,10 @@ include("helpers.jl")
 
 @timeit_include("compiletimes.jl")
 verify_no_running_processes()
+if get(ENV, "PLUTO_TEST_ONLY_COMPILETIMES", nothing) == "true"
+    show(TOUT; compact=true, sortby=:firstexec)
+    exit(0)
+end
```

Comment:
I've added this because it's very useful for local development. Just set the ENV var and run `pkg> test` to get the compilation times.

Reply:
Nice! Related: I use a script that only loads the relevant Pluto test files based on a given regex ([here](https://gist.github.com/Pangoraw/e1fb2db3c2d756f8c12ea70af2e3892e#file-testpluto-jl)). So that i can only `tpluto Macro|Notebook` This does not work with compiletimes since the print is at the bottom of runtests.jl tough.

Reply:
Great. I like it. Maybe I'll use your script in the future

Reply:
@Pangoraw Does it also print the total times in the end? I guess so right? Maybe it's actually a great idea to print it once after compiletimes and another once all the tests are done. That way people could also do

```julia
$ julia --project

julia> using TestEnv

julia> TestEnv.activate()

julia> include("test/helpers.jl");

julia> include("test/compiletimes.jl")

```

Reply:
the total time is 37s on my screenshot, not sure to which total it is refering tough because only this test file is run.